components/opponent/wikis/trackmania/opponent_display_custom.lua

```diff
@@ -0,0 +1,242 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:OpponentDisplay/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local DisplayUtil = require('Module:DisplayUtil')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+local PlayerDisplay = Lua.import('Module:Player/Display', {requireDevIfEnabled = true})
+
+local OpponentDisplayCustom = Table.deepCopy(OpponentDisplay)
+
+local html = mw.html
+
+--[[
+Display component for an opponent entry appearing in a bracket match.
+]]
+OpponentDisplayCustom.BracketOpponentEntry = Class.new(
+	function(self, opponent, options)
+		self.content = html.create('div'):addClass('brkts-opponent-entry-left')
+
+		if opponent.type == 'team' then
+			self:createTeam(opponent.template or 'tbd', options)
+		elseif opponent.type == 'solo' or opponent.type == 'duo' then
+			self:createPlayers(opponent)
+		elseif opponent.type == 'literal' then
+			self:createLiteral(opponent.name or '')
+		end
+
+		self.root = html.create('div'):addClass('brkts-opponent-entry')
+			:node(self.content)
+	end
+)
+
+function OpponentDisplayCustom.BracketOpponentEntry:createTeam(template, options)
```

Comment:
this function is identical with commons, right?

if so, then
`OpponentDisplayCustom.BracketOpponentEntry.createTeam = OpponentDisplay.BracketOpponentEntry.createTeam`
should be enough